src/views/ContinuationOut.vue

```diff
@@ -739,13 +752,8 @@ export default class ContinuationOut extends Mixins(CommonMixin, DateMixin,
       header.header.documentOptionalEmail = this.documentOptionalEmail
     }
 
-    switch (this.staffPaymentData.option) {
-      case StaffPaymentOptions.NO_FEE:
-        header.header['waiveFees'] = true
-        break
-
-      case StaffPaymentOptions.NONE: // should never happen
-        break
+    if (this.staffPaymentData.option === StaffPaymentOptions.NO_FEE) {
+      header.header['waiveFees'] = true
```

Comment:
Have another look at how Staff Payment Data is set in this filing. I was not expecting a check for NO_FEE here (it's hard-coded, right?).

Reply:
Yes, I think so. I'll have a look!

Reply:
Looks good.